posthog/api/survey.py

```diff
@@ -1356,7 +1365,7 @@ def get_surveys_response(team: Team):
 
 
 @csrf_exempt
-def surveys(request: Request):
+def surveys(request: Request, survey_id: str | None = None):
```

Comment:
Should we add a new endpoint instead to separate things and be better able to estimate costs in the future? (or even better, scale control whenever something happens)
like /external_survey/$surveyId?distinctId=123&...

Reply:
or, would we be able to estimate costs for requests with and w/o `survey_id`?

Reply:
now it's on a new endpoint, /external_surveys